.gitignore

```diff
@@ -0,0 +1,14 @@
+node_modules
```

Comment:
General Comment:
file names should always be dash-case (menu-icon instead of menuIcon, caption instead of Caption)... relevant both for folders and files.